src/main/groovy/io/openliberty/tools/gradle/tasks/DeployTask.groovy

```diff
@@ -328,18 +330,31 @@ class DeployTask extends AbstractServerTask {
         LooseEarApplication looseEar = new LooseEarApplication(task, config);
         looseEar.addSourceDir();
         looseEar.addApplicationXmlFile();
+        
+        //Checking ear plugin dependency configurations to determine loose-ear content
+        processDeployDependencies(looseEar, task)
+        processEarlibDependencies(looseEar, task)
+
+        File manifestFile = new File(project.buildDir.getAbsolutePath() + "/tmp/ear/MANIFEST.MF")
+        looseEar.addManifestFile(manifestFile)
+    }
+
+    private void processDeployDependencies(LooseEarApplication looseEar, Task task) {
+        HashMap<File, Dependency> completeDeployDeps = new HashMap<File, Dependency>();
 
         File[] filesAsDeps = task.getProject().configurations.deploy.getFiles().toArray()
-        Dependency[] deps = task.getProject().configurations.deploy.getAllDependencies().toArray()
-        HashMap<File, Dependency> completeDeps = new HashMap<File, Dependency>();
-        if(filesAsDeps.size() == deps.size()){
+        Dependency[] deployDeps = task.getProject().configurations.deploy.getAllDependencies().toArray()
+
+        if(filesAsDeps.size() == deployDeps.size()){
             for(int i = 0; i<filesAsDeps.size(); i++) {
-                completeDeps.put(filesAsDeps[i], deps[i])
+                completeDeployDeps.put(filesAsDeps[i], deployDeps[i])
             }
         }
 
-        logger.info(MessageFormat.format("Number of compile dependencies for " + task.project.name + " : " + completeDeps.size()))
-        for (Map.Entry<File, Dependency> entry : completeDeps){
+        task.getProject().configurations.earlib.getResolvedConfiguration().getFirstLevelModuleDependencies()
```

Comment:
Why is this statement here? The result is not assigned to a variable.

Reply:
Just a leftover from moving things around. I'll remove it.